templates/vue/src/main.js

```diff
@@ -0,0 +1,15 @@
+import Vue from 'vue'
+import App from './App.vue'
+
+Vue.config.productionTip = false
+
+import { initContract } from './utils'
+
+window.nearInitPromise = initContract()
+    .then(() => {
```

Comment:
this file should also use 2 space indents

Reply:
fixed